app/Integration/Academic/Mapper/FromArray/DefaultFromArrayMapper.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Integration\Academic\Mapper\FromArray;
+
+use Acme\Academic\Academic;
+use Acme\Academic\ValueObject\AcademicRegistrationNumber;
+use Acme\Article\ArticleCollection;
+use App\Integration\Article\Mapper\FromArray\FromArrayMapper as FromArrayArticleMapper;
+
+final class DefaultFromArrayMapper implements FromArrayMapper
```

Comment:
As commented on FromAcademicMapper, I think that's not a good name: It does not define the destination entity and does not explicit what kind of operation is done (yes, you just mention the source, not the target). This is an hydration, so what about **AcademicHidrator**?